docs/content/en/docs/workshops/provision/prod_cluster.md

```diff
@@ -0,0 +1,344 @@
+---
```

Comment:
Can we rename this file to "vsphere.md" As we get more providers and add them to this workshop it'll be confusing which one is considered "production"